packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -807,6 +807,36 @@ export default class Meetings extends WebexPlugin {
     );
   }
 
+  /**
+   * Creates a noise reduction effect
+   *
+   * @param {Object} options optional custom effect options
```

Comment:
would import the types from effect and remove the access token from it , we will need to be more specific when it comes to types 

same for noise

Reply:
+1 from me on this one, any new API we add has to be fully typed and I think we should also do something about jsdocs, because right now they require specifying types which is a duplication of the types already declared in typescript and it's a potential source of problems if there is a mismatch

Reply:
1. added the types for both the methods by removing `authToken` which we're explicitly passing to effects constructor.

2. for `jsdocs`, should we start not adding jsdocs to typescript files as anyway ts methods will annotate types on its own? It is recommended to use **jsdocs with js** and **typescript alone** but here we're doing **jsdocs with typescript** which is anyways not recommended.

we can discuss more if required on this topic.

Reply:
Agreed. Will discuss it with the team how we can proceed further on this. Will have a separate Jira for this.